packages/core/src/PhysicsManager.ts

```diff
@@ -0,0 +1,110 @@
+import { Ray, Vector3 } from "@oasis-engine/math";
+import { Layer } from "./Layer";
+import { ColliderFeature } from "./collider";
+import { Engine } from "./Engine";
+import { Collider } from "./collider/Collider";
+
+/**
+ * The result of raycast test.
```

Comment:
Should adjust comments, maybe this is not a ray exclusive class, `ShapeCast` is also applicable.